### ISSUE TITLE
Mean coverage and coverage variance

### ISSUE THREAD

**Issue by fitbio** [PapenfussLab/clove#8]
Hi,
I used gridss to find breakpoints. My question is which tool shall i use to calculate the mean coverage and coverage variance values from my bam file to use clove. It would be really helpful.

**Comment by lmfaber** [comment]
I don't think this is still relevant to you, but maybe this answer helps people in the future. I think you can use either of those options:

* Use [samtools coverage](http://www.htslib.org/doc/samtools-coverage.html) and calculate the values from the output.
   ```
   samtools coverage input.bam > coverage.txt
   mean_coverage=$(grep -v '^#' coverage.txt | awk 'BEGIN{OFS="\t"}{x+=$6}END{print x/NR}')
   sd_coverage=$(grep -v '^#' coverage.txt | awk 'BEGIN{OFS="\t"}{x+=6; y+=$6^2}END{print sqrt(y/NR-(x/NR)^2)}')
   ```
* I think you can also use [CollectWgsMetrics](https://broadinstitute.github.io/picard/command-line-overview.html#CollectWgsMetrics) from gatk/picard.
  Explanation of the output: [see here](https://broadinstitute.github.io/picard/picard-metric-definitions.html#CollectWgsMetrics.WgsMetrics).
  The metrics are called `MEAN_COVERAGE` and `SD_COVERAGE`

It would be nice if clove could detect these options itself, because one is already providing the bam file from which the information could be calculated. Looking at the activity of this repo, I think it's unlikely to happen, so stick to the above-mentioned options.
